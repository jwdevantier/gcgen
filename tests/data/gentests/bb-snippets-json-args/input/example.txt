# <<? hello
# ?>>

# <<? hello null
# ?>>

# <<? hello "Jane"
# ?>>

# <<? print_files ["/etc/passwd", "/etc/issue"]
# ?>>