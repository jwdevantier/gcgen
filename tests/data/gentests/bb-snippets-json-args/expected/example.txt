# <<? hello
hello stranger...
# ?>>

# <<? hello null
hello stranger...
# ?>>

# <<? hello "Jane"
hello Jane!
# ?>>

# <<? print_files ["/etc/passwd", "/etc/issue"]
printing file '/etc/passwd'...
printing file '/etc/issue'...
# ?>>